src/ConfigurationParser.cpp

```diff
@@ -95,6 +97,12 @@ Configuration ConfigurationParser::parse(
   configuration.usageCommand = extractCommand(p, USAGE_KEY);
 
   configuration.isDebugSet = getOrEmpty(p, DEBUG_KEY) == "true";
+
+  configuration.name = getOrEmpty(p, MODULE_NAME_KEY);
+  if (configuration.name.empty())
+    throw std::runtime_error("At least on module is missing mandatory " +
```

Comment:
error message is not clear to me. Maybe we could replace by: "Module name is missing for + MODULE_NAME_KEY

Reply:
`MODULE_NAME_KEY` is just the name of the parameter. What would be nice is to be able to point which module (isolator index for example) is concerned, but I wasn't able to extract this information (that's the whole point of naming explicitly modules). So to me we can't do much more that warn the user that (at least) one specified module doesn't have a name.

Reply:
I suggest the following message:

> "At least one module is missing '" + MODULE_NAME_KEY + "' parameter."